tendrl/commons/objects/cluster/atoms/configure_monitoring/__init__.py

```diff
@@ -90,10 +98,12 @@ def run(self):
                 node_plugin,
                 plugin_params
             )
-        plugin_config_success &= self._configure_plugin(
-            'tendrl_%s' % NS.tendrl_context.sds_name,
-            plugin_params
-        )
+        if NS.tendrl_context.sds_name == 'gluster':
+            for gluster_plugin in GLUSTER_CLUSTER_PLUGINS:
+                plugin_config_success &= self._configure_plugin(
+                    gluster_plugin,
+                    plugin_params
+                )
```

Comment:
So you were missing on configuring many plugins before this PR it seems from the code change?